### ISSUE TITLE
ttnn::pad only supports padding on the lowest 3 dimensions for tensors with rank > 4

### ISSUE THREAD

**Issue by djstefanovicTT** [tenstorrent/tt-metal#38144]
ttnn::pad only supports padding on the lowest 3 dimensions for tensors with rank > 4. Ideally it should support all dimensions. 
Following error occurs:
Always | TT_FATAL: ttnn::pad only supports padding on the lowest 3 dimensions for tensors with rank > 4

Python script to reproduce the issue:
```
import ttnn
import torch

device = ttnn.open_device(device_id=0)

input_torch = torch.randn(1, 4, 4, 4, 128, dtype=torch.bfloat16)
input_ttnn = ttnn.from_torch(
    input_torch, dtype=ttnn.DataType.BFLOAT16, layout=ttnn.Layout.ROW_MAJOR, device=device
)

padding = [(0, 0), (1, 1), (0, 0), (0, 0), (0, 0)]
output_ttnn = ttnn.pad(input_ttnn, padding, 0.0)

print(output_ttnn)

ttnn.close_device(device)
```

**Comment by skhatriTT** [comment]
@fahmedchanTT , could you please provide an update on when this issue is expected to be scheduled?

**Comment by sdjordjevicTT** [comment]
Is there some update regarding this?
